scripts/20-setup-sj201.sh

```diff
@@ -48,6 +52,7 @@ for k in "${kernels[@]}"; do
     make -j${1:-} all || exit 2
     mkdir -p "/lib/modules/${kernel}/kernel/drivers/vocalfusion"
     cp vocalfusion* "/lib/modules/${kernel}/kernel/drivers/vocalfusion" || exit 2
+    cp ../*.dtbo /boot/overlays/
```

Comment:
Why was this added?

Reply:
This file is included in the overlay, but it is probably better to include the built version here in case the drivers are updated